### PR TITLE
Adding additional reboot (optional)

### PR DESCRIPTION
**Pull request by automationxpert**
# Proposed Changes
- Added playbook reboot.yml for rebooting servers
- Added reboot.sh to execute reboot.yml easily just like deploy.sh and reset.sh

## Checklist

- [x] Tested locally
- [x] Ran `site.yml` playbook
- [x] Ran `reset.yml` playbook
- [x] Did not add any unnecessary changes
- [x] 🚀
- [x] Ran pre-commit install at least once before committing

### CODE REVIEW

File: reboot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+- name: Linux Reboot Demo
+  hosts: k3s_cluster
+  gather_facts: yes
+  become: yes
+  tasks:
+    - name: Reboot the machine (Wait for 5 min)
+      reboot:
+        reboot_timeout: 300
```
Comment (timothystewart6) [inline reboot.yml:9]: I think this should be `3000` (300 seconds === 5 minutes)

File: reboot.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+- name: Linux Reboot Demo
```
Comment (timothystewart6) [inline reboot.yml:2]: Any way we can name this something like `Reboot k3s_cluster`, rather than demo?  Thank you!

Comment (automationxpert) [reply reboot.yml]: Yeah, That would be better!!! Just updated the name. Thank You :)

Comment (timothystewart6) [reply reboot.yml]: Thank you so much!

**Pull request merged**